lib/ansible/modules/cloud/azure/azure_rm_image.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+author: Emma Laurijssens van Engelenhoven
+description:
+  - Capture an Azure Virtual Machine image for the deployment of other VMs in Azure
+     The VM should be generalized using sysprep. After this command runs, the Virtual Machine will be unusable.
+     Depends on pip azure module 2.0.0 or above and azure-mgmt-compute 2.1.0 and above.
+module: azure_rm_image
+options:
+  resource_group_name:
```

Comment:
Looks like this should be just `resource_group` to align with the arg spec and the other Azure modules.